pom.xml

```diff
@@ -476,6 +476,36 @@
             <plugins.docker>docker</plugins.docker>
          </properties>
       </profile>
+      
+      <profile>
+         <id>site</id>
+         <activation>
+            <property>
+               <name>make-site</name>
+            </property>
```

Comment:
I just finished rewriting Building binaries page to include build parameters, this one included, coming soon.